node/src/indexer/lib.rs

```diff
@@ -60,3 +71,18 @@ pub(crate) async fn get_mpc_contract_state(
         }
     }
 }
+
+pub(crate) async fn get_mpc_contract_state(
+    mpc_contract_id: AccountId,
+    client: &actix::Addr<near_client::ViewClientActor>,
+) -> anyhow::Result<(u64, ProtocolContractState)> {
+    get_mpc_state(mpc_contract_id, client, CONTRACT_STATE_ENDPOINT).await
+}
+
+#[cfg(feature = "tee")]
+pub(crate) async fn get_mpc_tee_state(
```

Comment:
Seems like this method doesn't get the full TEE state. Should we rename it?

```suggestion
pub(crate) async fn get_mpc_allowed_image_hashes(
```